apis/fluent-en_GB/atrium-api-fluent-en_GB-jvm/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/PathAssertionsSpec.kt

```diff
@@ -20,8 +21,15 @@ class PathAssertionsSpec : ch.tutteli.atrium.specs.integration.PathAssertionsSpe
     fun0(Expect<Path>::isRegularFile),
     fun0(Expect<Path>::isDirectory),
     fun1(Expect<Path>::hasSameBinaryContentAs),
-    fun3(Expect<Path>::hasSameTextualContentAs)
+    fun3(Expect<Path>::hasSameTextualContentAs),
+    fun1(Companion::hasSameTextualContentAsDefaultArgs)
 ) {
+
+    companion object : WithAsciiReporter() {
```

Comment:
please remove WithAsciiReporter, this is only used in the infix API

Reply:
can you give suggestion how to can I call hasSameTextualContentAs method without companion object (hasSameTextualContentAs has 3 parameters)

Reply:
Keep the companion, just remove WithAsciiReporter